scholly/base/src/layouts/layout1/components/layout/Header.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * This file exports header for Scholly (client) page.
+ */
+import React from 'react';
+import { Container, Nav, Navbar } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { TEAM_INFO } from '../../../../global/data';
+import { getRoutes } from '../router/routes';
+
+const Header = () => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Make sure to add a TODO somewhere (e.g. Trello) to fix this warning and add the twitter component back in.

Reply:
noted!